README.md

```diff
@@ -61,6 +61,7 @@ the environment.
 | namespace | K8s namespace used to deploy endpoints and services | `string` | `"production"` | no |
 | network | Authorized network allowed to communicate with redis instance | `string` | `"default"` | no |
 | project | Name of GCP project | `string` | n/a | yes |
+| redis\_version | Redis instance version. Must be on of these values: REDIS\_5\_0, REDIS\_4\_0, REDIS\_3\_2 | `string` | `"REDIS_4_0"` | no |
```

Comment:
Tady bych místo těch vyjmenovaných hodnot hodila spíš link na https://www.terraform.io/docs/providers/google/r/redis_instance.html#redis_version, aby se nestalo, že ten seznam budeme mít nekompletní, až přidají novou verzi. To samé pak dole v description té proměnné.